lib/dblock.php

```diff
@@ -4,19 +4,14 @@
 
 use OCP\IConfig;
 use OCP\IDb;
+use OCP\IDBConnection;
```

Comment:
I think we can remove both lines, because we don't need a database connection at this point.